brain/cell/morphologyImpl.cpp

```diff
@@ -0,0 +1,252 @@
+
+/* Copyright (c) 2013-2015, EPFL/Blue Brain Project
+ *                          Juan Hernando <jhernando@fi.upm.es>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "morphologyImpl.h"
+
+#include <brion/morphology.h>
+
+#include <lunchbox/log.h>
+
+#include <boost/foreach.hpp>
+
+#include <bitset>
+
+namespace brain
+{
+namespace cell
+{
+
+Morphology::Impl::Impl( const brion::Morphology& morphology )
```

Comment:
No, no. Morphology::Impl is the class name the other Impl is the name of the method, which is the constructor.